setup.py

```diff
@@ -1,6 +1,6 @@
 try:
     from overlays import builder
-    #builder.compile()
+    builder.compile()
     builder.copy()
 except:
     pass
```

Comment:
Could we print something here? I tried ran setup.py without installing dtc (just to see how it would handle it), and I was surprised that it didn't complain.
Passing right over the failure is good for folks who are not trying to use overlays, but it could be confusing for someone who is trying to use overlays but didn't read your readme instructions about installing dtc.